pages/troubleshooting/faq.md

```diff
@@ -26,6 +27,9 @@ title: FAQs
 ##### Can I use multiple containers?
 We are planning, and committed, to adding support for having multiple apps/containers running on a device. While the work towards multiple apps is in progress, as an interim solution, we do however  have a few users running [multiple containers within an app via docker-compose](https://resin.io/blog/multi-container-with-docker-compose-on-resin-io/) and [have done work with kubernetes](https://resin.io/blog/our-first-experiments-with-multi-container-apps/) in the same fashion.
 
+##### How do I push a new git repo to an Application?
+If you have pushed a repository called `project-A` to your application (called `myapp`) and at a later stage you would like to push a new project called `project-B`, you can do this by adding the application remote (`git remote add resin joesoap@git.resin.io:joesoap/myapp.git`) to `project-B`'s local repository. You can then easily push `project-B` to `myapp` by just doing `git push resin master -f`. The extra `-f` on the command forces the push and resets the git history on the git remote on resin.io's backend. You should now have `project-B` running on all the devices in the `myapp` fleet.
```

Comment:
Would add a line that _after this_ one should normally not use `-f` anymore and do the regular pushes.